src/Magento_Swatches/web/js/swatch-renderer-ext.js

```diff
@@ -32,6 +32,9 @@ define(['jquery', 'underscore', 'mage/translate'], function($, _, $t) {
                     return;
                 }
 
+                this.options.selectorProduct =
+                    '.product-info-main, .cs-product-tile';
+
```

Comment:
Please move the fix a bit up to widget options. It will be more flexible to place proper value for `selectorProduct` there.

Reply:
I'm afraid that won't work, because `selectorProduct` is an already defined option? this line is here to replace the existing option value. 

At least somehow I think it didn't work in our case if I just added it next to other options